api/src/main/java/org/openmrs/module/fhir2/api/util/FhirUtils.java

```diff
@@ -0,0 +1,52 @@
+/*
```

Comment:
We should probably consider renaming either this file to `FhirReferenceUtils` or the file in the omod to `FhirServerUtils` (I favour the second of these) just to avoid confusion.